python/cudf/cudf/utils/ioutils.py

```diff
@@ -239,11 +266,11 @@
     If ``True``, write timestamps in int96 format. This will convert
     timestamps from timestamp[ns], timestamp[ms], timestamp[s], and
     timestamp[us] to the int96 format, which is the number of Julian
-    days and the number of nanoseconds since midnight. If ``False``,
-    timestamps will not be altered.
-row_group_size_bytes: integer or None, default None
+    days and the number of nanoseconds since midnight of 1970-01-01.
+    If ``False``, timestamps will not be altered.
+row_group_size_bytes: integer, default {row_group_size_bytes_val}
     Maximum size of each stripe of the output.
-    If None, 134217728 (128MB) will be used.
+    If None, {row_group_size_bytes_val} (128MB) will be used.
```

Comment:
This formatted value isn't guaranteed to stay in sync with the text `(128MB)`. We might remove the parenthesized text, or make it match. Something like:
```suggestion
    If None, {row_group_size_bytes_val} ({row_group_size_bytes_val/1024/1024} MB) will be used.
```